projects/igniteui-angular/src/lib/buttonGroup/buttonGroup.component.ts

```diff
@@ -298,10 +298,6 @@ export class IgxButtonGroupComponent extends DisplayDensityBase implements After
         const button = this.buttons[index];
         const buttonElement = button.nativeElement;
 
-        if (buttonElement.classList.contains('igx-button--disabled')) {
```

Comment:
This should be removed from deselectButton method also, because now when have single selection the selection of disabled button is not removed